American Whitewater/ViewControllers/Map/MapViewController.swift

```diff
@@ -28,14 +32,31 @@ class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerD
         locationManager.desiredAccuracy = kCLLocationAccuracyHundredMeters
         
         self.locationManager.delegate = self
+        
+        updateFilterButton()
+        fetchReachesFromCoreData()
+        
+        notificationObservers.append(
```

Comment:
You know I'm not the biggest fan of notifications, but this seems like a reasonable use case.

Reply:
👍🏼 I was thinking about adopting Combine and making the properties on DefaultsManager @Published. But that's a bigger change and in any event isn't an option til after iOS 13+ is merged.